inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1311,3 +1310,16 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
 
     return newNode;
 }
+
+bool MKLDNNNode::canBePerformedAsScaleShift() const {
+    bool inputsIsConst = true;
+    for (size_t i = 1; i < getOriginalInputsNumber(); i++) {
+        if (!getParentEdgesAtPort(i)[0]->getParent()->isConstant() || getParentEdgesAtPort(i)[0]->getParent()->getType() != Input) {
```

Comment:
At this moment decomposition tests failed due to call ```isConstant()```. I believe it's not related to current changes. I investigate this problem.

Reply:
Problem was that we call ```isConstant()``` before fusing and ```const``` type was assigned for node but after fusing real type was ```noConst```